assembl/static2/js/app/components/harvesting/harvestingBox.jsx

```diff
@@ -0,0 +1,213 @@
+// @flow
+import React from 'react';
+import ARange from 'annotator_range'; // eslint-disable-line
+import { connect } from 'react-redux';
+import { compose, graphql } from 'react-apollo';
+import { Button } from 'react-bootstrap';
+import { Translate } from 'react-redux-i18n';
+import classnames from 'classnames';
+import moment from 'moment';
+
+import addPostExtractMutation from '../../graphql/mutations/addPostExtract.graphql';
+import withLoadingIndicator from '../../components/common/withLoadingIndicator';
+import { displayAlert } from '../../utils/utilityManager';
+import { getConnectedUserId, getConnectedUserName } from '../../utils/globalFunctions';
+import AvatarImage from '../common/avatarImage';
+
+type Props = {
+  extract: ?Extract,
+  postId: string,
+  contentLocale: string,
+  selection: ?Object,
```

Comment:
please describe the shape of this object if possible

Reply:
@cedricmessiant I don't know how to describe this object. It's corresponds to "window.getSelection()" :(

Reply:
Also: we use its methods, not its properties.